template/koa-rest-api/src/app.ts

```diff
@@ -1,9 +1,11 @@
-import 'skuba-dive/register';
+import './register';
 
-import { router } from 'src/api';
-import { config } from 'src/config';
-import { createApp } from 'src/framework/server';
+import { router } from './api';
+import { config } from './config';
+import { createApp } from './framework/server';
 
 const app = createApp(router.allowedMethods(), router.routes());
 
-export default Object.assign(app, { port: config.port });
+Object.assign(app, { port: config.port });
+
+export default app;
```

Comment:
Inconsequential change, but I tried to make it a bit more obvious that `Object.assign` is mutating here.